### PR TITLE
Kkraune/schema

### PR DESCRIPTION
**Pull request by kkraune**
The schema guide is now more high level and with less irrelevant text

### CODE REVIEW

**Comment by kkraune** [comment]
@hmusum please review

File: en/schemas.html [hunk 1]
```diff
@@ -257,5 +255,32 @@ <h2 id="content-cluster-mapping">Content cluster mapping</h2>
...
+</pre>
+<p>
+  To accept such changes, add a <a href="reference/validation-overrides.html">validation-override</a>, e.g.
+</p>
+<pre>
+&lt;validation-overrides&gt;
+    &lt;allow until="2021-08-30"&gt;indexing-change&lt;/allow&gt;
+&lt;/validation-overrides&gt;
+</pre>
+<p>
```
Comment (hmusum) [inline en/schemas.html]: I don't think a change leading to a restart is a good example, the change will happen anyway and there is no validation override for restart

**Review by hmusum — CHANGES_REQUESTED** [review]

File: en/schemas.html (hunk 1 above)
Comment (kkraune) [reply en/schemas.html]: Good point! modifying to a field indexing change instead

**Comment by kkraune** [comment]
@hmusum  ptal

**Review by hmusum — APPROVED** [review]
Great!

**Pull request merged**